wireguard-install.sh

```diff
@@ -89,27 +173,20 @@ if [[ $1 == "add-client" ]]; then
 		echo "Please install and configure WireGuard first."
 		exit 1
 	fi
-elif [[ -e /etc/wireguard/params ]]; then
-	echo "WireGuard is already installed. Run with 'add-client' to add a client."
-	exit 1
-fi
-
-# Check OS version
-if [[ -e /etc/debian_version ]]; then
-	source /etc/os-release
-	OS=$ID # debian or ubuntu
-elif [[ -e /etc/fedora-release ]]; then
-	source /etc/os-release
-	OS=$ID
-elif [[ -e /etc/centos-release ]]; then
-	OS=centos
-elif [[ -e /etc/arch-release ]]; then
-	OS=arch
-else
-	echo "Looks like you aren't running this installer on a Debian, Ubuntu, Fedora, CentOS or Arch Linux system"
+elif [[ $1 == "uninstall" ]]; then
+	if [[ -e /etc/wireguard ]]; then
+		uninstall
+        	exit 0
+	else
+		echo "WireGuard is not installed."
+		exit 1
+	fi
+elif [[ -e /etc/wireguard ]]; then
```

Comment:
The path is `/etc/wireguard/params`

Reply:
If we can detect /etc/wireguard directory itself, instead of specific file inside it, I suppose that should be more accurate. Plz suggest

Reply:
I see it was changed in previous commits. Corrected.